Taskfile.yml

```diff
@@ -120,15 +120,15 @@ tasks:
       - '{{.OFFLINETEST_BIN_RELEASE_DIR}}/offline-test --iso_data_path "./iso_data/{{.GAME}}" --game {{.GAME}}'
   offline-tests-fast:
     cmds:
-      - '{{.OFFLINETEST_BIN_RELEASE_DIR}}/offline-test --iso_data_path "./iso_data/{{.GAME}}" --game {{.GAME}} --dump_current_output --num_threads 10'
+      - '{{.OFFLINETEST_BIN_RELEASE_DIR}}/offline-test --iso_data_path "./iso_data/{{.GAME}}" --game {{.GAME}} --dump_current_output --num_threads 32'
```

Comment:
kinda unrelated, but if we start adding lots of dgos, there's a lot of duplicated work/memory usage because each thread will end up reading all the dgos

Reply:
Yeah I somewhat expected that tradeoff, do you think it's something that could be easily improved though?  Or perhaps when we have many dgos i can try to partition the work by the dgos themselves so there isn't somewhat-random access within a single thread.

Reply:
yeah, I think we can either partition by dgo, or share the ObjectFileDB somehow.

I tried running jak 1 with 16 threads and very quickly ran out of ram